### PR TITLE
Made Results card responsive to the device size

### PR DESCRIPTION
**Pull request by tapaswini-kodavanti**

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -19,14 +19,20 @@ input {
...
+}
+
 .Results {
   background-color: white;
-  margin-left: 150px;
-  margin-right: 150px;
-  margin-top: 20px;
+  margin-left: auto;
+  margin-right: auto;
+  margin-top: 5%;
```
Comment (utk09) [inline src/App.css]: Really clever solution, like it.

But on mobile phones, the card becomes smaller and the search bar is little too narrow. Can we please add `@media queries` to accommodate for various screen sizes?

Along with that, I think an appropriate solution would also be to cater all elements that are going to come on the page - like map and news cards. I would suggest reaching out to fellows who are working on these items and see what page size would be perfect for all elements to come on the page without writing repetitive CSS.

File: example.env [hunk 2]
```diff
@@ -1 +1 @@
-REACT_APP_APIKEY=
\ No newline at end of file
+REACT_APP_APIKEY=""
```
Comment (utk09) [inline example.env]: Does this need to be empty inverted comma? Can we remove it?

**Review by utk09 — CHANGES_REQUESTED** [review]
Good work - please check the review comments

**Comment by isalyu** [comment]
(1) This is very picky; and maybe this wouldn't look better, but to me (again this a design comment), I think the input bar feels a little wider than needed in full screen mode (on laptop)

**Comment by utk09** [comment]
> (1) This is very picky; and maybe this wouldn't look better, but to me (again this a design comment), I think the input bar feels a little wider than needed in full screen mode (on laptop)

Please do implement this @tapaswini-kodavanti

**Pull request merged**